### PR TITLE
Update with the recent dependencies updates.

### PR DESCRIPTION
**Pull request by SPalominos**
Update with the recent dependencies updates.
This PR take the history of the dev branch in order to keep the improvement made on the branch.

### CODE REVIEW

File: geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy [hunk 1]
```diff
@@ -666,7 +666,7 @@ IProcess applyRandomForestModel() {
             def df = dfNofactorized
             // Identify columns being string (thus needed to be factorized)
             inputColumns.each{colName, colType ->
-                if(colType == "STRING" || colType=="VARCHAR"){
+                if(colType == "STRING" || colType=="VARCHAR" || colType=="VARCHAR"){
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy:669]: CHARACTER VARYING

File: geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/RsuIndicators.groovy [hunk 2]
```diff
@@ -1239,7 +1239,7 @@ return create {
                 def roadTable_zindex0_buffer = postfix "road_zindex0_buffer"
                 def road_tmp = postfix "road_zindex0"
                 datasource """DROP TABLE IF EXISTS $roadTable_zindex0_buffer, $road_tmp;
-            CREATE TABLE $roadTable_zindex0_buffer as SELECT st_buffer(the_geom, WIDTH::double precision/2)
+            CREATE TABLE $roadTable_zindex0_buffer as SELECT st_buffer(the_geom, WIDTH::DOUBLE/2)
```
Comment (ebocher) [inline geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/RsuIndicators.groovy:1242]: Why remove double precision?

File: processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/GeoIndicatorsChainTest.groovy [hunk 3]
```diff
@@ -149,6 +150,7 @@ class GeoIndicatorsChainTest {
     }
 
     @Test
+    @Disabled
```
Comment (ebocher) [inline processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/GeoIndicatorsChainTest.groovy:153]: This test doesn't work any more ?

File: geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy (hunk 1 above)
Comment (SPalominos) [reply geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/TypologyClassification.groovy]: Fixed

File: geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/RsuIndicators.groovy (hunk 2 above)
Comment (SPalominos) [reply geoindicators/src/main/groovy/org/orbisgis/orbisprocess/geoclimate/geoindicators/RsuIndicators.groovy]: Fixed

File: processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/GeoIndicatorsChainTest.groovy (hunk 3 above)
Comment (SPalominos) [reply processingchain/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/processingchain/GeoIndicatorsChainTest.groovy]: The test was disabled on the dev branch.

**Comment by ebocher** [comment]
Excellent !

**Pull request merged**